src/test/resources/expected/test-attribute-1.kore.golden

```diff
@@ -1,28 +1 @@
-Definition
-    { definitionAttributes = Attributes []
-    , definitionModules =
-        [ Module
-            { moduleName = ModuleName "TEST-ATTRIBUTE-1"
-            , moduleSentences =
-                [ SentenceSortSentence SentenceSort
-                    { sentenceSortName = Id Object "Nat"
-                    , sentenceSortParameters = []
-                    , sentenceSortAttributes =
-                        Attributes 
-                            [ MetaPattern (ApplicationPattern Application
-                                { applicationSymbolOrAlias =
-                                    SymbolOrAlias
-                                        { symbolOrAliasConstructor = Id Meta "#hook"
-                                        , symbolOrAliasParams = []
-                                        }
-                                , applicationChildren =
-                                    [ MetaPattern (StringLiteralPattern (StringLiteral "Nat"))
-                                    ]
-                                })
-                            ]
-                    }
-                ]
-            , moduleAttributes = Attributes []
-            }
-        ]
-    }
\ No newline at end of file
+Parse error: Error in module 'TEST-ATTRIBUTE-1' -> sort 'Nat' declaration -> attributes -> symbol or alias '#hook': Symbol '#hook' not defined..
```

Comment:
I'm not sure this test was supposed to fail... maybe the original .kore file needs to be changed? @hreada

Reply:
@traiansf I've examined the original .kore file, and there's no problem there. This test should not fail. I want to know if there was an error in parsing it, or in checking its well-formedness. We should not check the well-formedness of patterns in attributes using the declarations in the module where the attributes are in. For now, it is OK to not checking well-formedness of patterns that are in attributes.

Reply:
Fixed by not checking attribute welformednees.